packages/switch/stories/index.stories.mdx

```diff
@@ -81,7 +81,8 @@ Simple example that illustrates where validation feedback will be displayed.
         return [...super.validationTypes, 'info'];
       }
     }
-    customElements.define('custom-switch', CustomSwitch);
+    if (!customElements.get('custom-switch'))
```

Comment:
I guess the linter doesn't lint javascript inside mdx but we try to use { } for if statement bodies even if they are single statements and can do without it. Mostly for readability.

And usually I would say this guard is a bad practice because not getting the error when a 2nd custom-switch is trying to define is potentially more harmful than getting the error, but for a custom-switch inside a demo, purely for demo purposes, I think it's fine :).

Reply:
Thanks for the feedback! I added the `{ }` for the if statement.